_posts/2021-02-16-release-1.12.2.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title:  "Apache Flink 1.12.2 Released"
+date:   2021-02-16 00:00:00
+categories: news
+authors:
+- yuan:
+  name: "Yuan Mei"
+- roman:
+  name: "Roman Khachatryan"
+
+---
+
+The Apache Flink community released the next bugfix version of the Apache Flink 1.12 series.
+
+This release includes 69 fixes and minor improvements for Flink 1.12.1. The list below includes a detailed list of all fixes and improvements.
```

Comment:
JIRA shows 68 issues resolved in 1.12.2.
https://issues.apache.org/jira/projects/FLINK/versions/12349502

Reply:
Updated to 83 issues (RC2), thanks!